lib/matplotlib/image.py

```diff
@@ -419,14 +428,18 @@ def _make_image(self, A, in_bbox, out_bbox, clip_bbox, magnification=1.0,
                 # colormapping works correctly
                 hid_output = output
                 output = np.ma.masked_array(
-                    hid_output[..., 0], hid_output[..., 3] < 0.5)
+                    hid_output[..., 0], hid_output[..., 3] == 0)
```

Comment:
I think this is going to break a test, but may fix an issue that @QuLogic found with imshow after #8024